.yamato/publish.yml

```
@@ -9,12 +9,12 @@ publish:
     image: package-ci/win10:stable
     flavor: b1.large
   commands:
-    - npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
+    - npm install upm-ci-utils@stable -g --registry {{ upmci_registry }}
     - upm-ci package publish --package-path com.unity.perception
   triggers:
     tags:
       only:
-        - /^(r|R)(c|C)-\d+\.\d+\.\d+(-preview(\.\d+)?)?$/
+        - "/^(r|R)(c|C)-+(-preview(+)?)?$/"
```

This still doesn't look like the proper regex. The original one looks right to me.
```suggestion
        - /^(r|R)(c|C)-\d+\.\d+\.\d+(-preview(\.\d+)?)?$/
```